opwen_email_server/actions.py

```diff
@@ -32,7 +35,7 @@
 Response = Union[dict, Tuple[str, int]]
 
 
-class _Action(ABC, LogMixin):
+class Action(ABC, LogMixin):
```

Comment:
Given that this is no longer required outside of this module, we can once again make this private by renaming it to `_Action`.